test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+import sys
+import torch
+from torch._C import parse_schema
+
+
+def check_bc(new_schema_dict):
+    existing_schemas = torch._C._jit_get_all_schemas()
+    for existing_schema in existing_schemas:
+        print("processing existing schema: ", str(existing_schema))
+        new_schemas = new_schema_dict.get(existing_schema.name, [])
+        found = False
+        for new_schema in new_schemas:
+            if new_schema.is_backward_compatible_with(existing_schema):
+                found = True
+                break
+        if not found:
+            print('Can NOT find backward compatible schemas after changes'
```

Comment:
nit: need a space here, it shows up as 'changesfor'

Reply:
good catch